data/class/pages/upgrade/LC_Page_Upgrade_ProductsList.php

```diff
@@ -220,4 +221,20 @@ public function process()
             return;
         }
     }
+
+    /**
+     * ホストの dtb_module に基づいた本当のインストール有無状態を取得し、
+     * オーナーズストアからの installed_flg を上書きする。
+     * @param array $productData
+     * @return void
+     */
+    private function detectInstalledFlagByHostState(&$productData)
+    {
+        $productId = $productData['product_id'];
+        $objQuery = \SC_Query_Ex::getSingletonInstance();
+
+        $isInstalled = $objQuery->exists('dtb_module', 'module_id = ?', array($productId));
```

Comment:
`del_flg = 0` もチェックした方が良さそうですがいかがでしょう？

```suggestion
        $isInstalled = $objQuery->exists('dtb_module', 'module_id = ? AND del_flg = 0', array($productId));
```